mycore-acl/src/main/java/org/mycore/mcr/acl/accesskey/restapi/v2/MCRRestDerivateAccessKeys.java

```diff
@@ -74,18 +82,18 @@ public class MCRRestDerivateAccessKeys {
     @Operation(
         summary = "Lists all access keys for a derivate",
         responses = {
-            @ApiResponse(responseCode = "200",
+            @ApiResponse(responseCode = OK,
                 description = "List of access keys attached to this derivate",
                 content = { @Content(mediaType = MediaType.APPLICATION_JSON,
                     array = @ArraySchema(schema = @Schema(implementation = MCRAccessKey.class))) }),
-            @ApiResponse(responseCode = "" + MCRObjectIDParamConverterProvider.CODE_INVALID, // 400
+            @ApiResponse(responseCode = BAD_REQUEST,
                 description = MCRObjectIDParamConverterProvider.MSG_INVALID,
                 content = { @Content(mediaType = MediaType.APPLICATION_JSON) }),
-            @ApiResponse(responseCode = "401",
-                description = "You do not have create permission and need to authenticate first",
+            @ApiResponse(responseCode = UNAUTHORIZED,
+                description = MCRRestDescription.UNAUTHORIZED,
                 content = { @Content(mediaType = MediaType.APPLICATION_JSON) }),
-            @ApiResponse(responseCode = "404",
-                description = "Derivate or access key does not exist",
+            @ApiResponse(responseCode = NOT_FOUND,
+                description = MCRRestDescription.DERIVATE_NOT_FOUND,
```

Comment:
`MCRRestDescription.DERIVATE_NOT_FOUND` is not equivalent to _Derivate or access key does not exist_.
Create new local member.